src/overlays/actors/ovl_En_Fall2/z_en_fall2.c

```diff
@@ -15,7 +15,14 @@ void EnFall2_Destroy(Actor* thisx, PlayState* play);
 void EnFall2_Update(Actor* thisx, PlayState* play);
 void EnFall2_Draw(Actor* thisx, PlayState* play);
 
-#if 0
+void func_80C1B8A4(EnFall2*, PlayState*);
+void func_80C1B9D4(EnFall2*, PlayState*);
+
+extern void* D_06008898;
+extern void* D_06005EF4;
+extern void* D_06008840;
+extern void* D_06005F10;
```

Comment:
For these, you can use the symbols in object_fall2 instead of using externs like this. Just `#include` the object_fall2.h header to access them; you can see an example of this in EnFall. For D_06008840, you can use object_fall2_Matanimheader_008840. For the other three, you'll have to make your own blobs; just ask on Discord if you need help doing that.